knexfile.js

```diff
@@ -10,28 +30,11 @@ module.exports = {
       password: process.env.DB_PASS,
       database: process.env.DB,
     },
-
     migrations: {
```

Comment:
I would clean these up! I  have no clue what they should look like. They're not breaking anything right now, but notice on lines 18 - 23 and lines 8 - 13 we have duplicate code.